libs/novui/src/components/select/Select.tsx

```diff
@@ -1 +1,39 @@
-export { Select } from '@mantine/core';
+import { Loader, Select as ExternalSelect } from '@mantine/core';
+import { forwardRef } from 'react';
+import { IconArrowDropDown } from 'src/icons';
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** there are imports of src/.. instead of ../icons/.. , it fails to build :)‏